app/src/core/MainClient.ts

```diff
@@ -50,116 +50,33 @@ export default class MainClient extends EventEmitter {
         this._request = new Request(this._catchErrors);
 
         this._profilesController = new ProfilesController(this._request, this._dispatch);
+        this._mainApi = new MainApi(this._request, this._socket, this._dispatch, this._profilesController);
 
         this._bindProfileListeners();
         this._bindCatchErrorsListeners();
     }
 
-    public catchErrors(error: string) {
+    get mainApi() {
+        return this._mainApi;
+    }
+
+    catchErrors(error: string) {
         this._catchErrors.catch(error);
     }
 
     // TODO Удалить после рефакторинга звонков (useWebRTC)
-    public getSocket() {
```

Comment:
you delete "public" because it default, right?

Reply:
yes, "public" is default modifier for class methods